apps/project/firebase.py

```diff
@@ -0,0 +1,96 @@
+import logging
+import typing
+
+from celery import shared_task
+from django.utils import timezone
+from firebase_admin.db import Reference as FbReference
+from pyfirebase_mapswipe import models as firebase_models
+from pyfirebase_mapswipe import utils as firebase_utils
+
+from apps.project.models import FirebasePushStatusEnum, Organization
+from main.config import Config
+from main.logging import log_extra
+
+logger = logging.getLogger(__name__)
+
+
+class InvalidOrganizationPushException(Exception): ...
+
+
+def handle_new_organization_on_firebase(organization: Organization, organization_ref: FbReference):
+    organization_data = firebase_models.FbOrganisation(
+        name=organization.name,
+        nameKey=organization.unique_name,
```

Comment:
@tnagorra Do we need nameKey? or What could be the value for the nameKey?

```suggestion
        nameKey="",
```